### PR TITLE
Add 4.11 latest version to update-version

### PR DESCRIPTION
**Pull request by petrkotas**
### Which issue this PR addresses:

Fixes

### What this PR does / why we need it:
Adds 4.11 version to the aro update-versions.

### CODE REVIEW

File: pkg/util/version/const.go [hunk 1]
```diff
@@ -31,6 +31,13 @@ var InstallStream = &Stream{
 	PullSpec: "quay.io/openshift-release-dev/ocp-release@sha256:b9fad814fb4442e7e852b0614d9bb4e2ebc5e1a2fa51623aa838b4ee0e4a5369",
 }
 
+var AdditionalStreams = map[string]*Stream{
+	"4.11.26": {
+		Version:  NewVersion(4, 11, 26),
+		PullSpec: "quay.io/openshift-release-dev/ocp-release@sha256:1c3913a65b0a10b4a0650f54e545fe928360a94767acea64c0bd10faa52c945a"
```
Comment (s-amann) [inline pkg/util/version/const.go:37]: please add a comma 
```suggestion
		PullSpec: "quay.io/openshift-release-dev/ocp-release@sha256:1c3913a65b0a10b4a0650f54e545fe928360a94767acea64c0bd10faa52c945a",
```

**Review by s-amann — CHANGES_REQUESTED** [review]
there's a need for a comma below

File: pkg/util/version/const.go (hunk 1 above)
Comment (petrkotas) [reply pkg/util/version/const.go]: AH I missed it. Thank you!

**Review by s-amann — APPROVED** [review]

File: pkg/util/version/const.go [hunk 2]
```diff
@@ -31,6 +31,13 @@ var InstallStream = &Stream{
 	PullSpec: "quay.io/openshift-release-dev/ocp-release@sha256:b9fad814fb4442e7e852b0614d9bb4e2ebc5e1a2fa51623aa838b4ee0e4a5369",
 }
 
+var AdditionalStreams = map[string]*Stream{
```
Comment (bennerv) [inline pkg/util/version/const.go:34]: Why use a map?  Why not a slice?

**Pull request merged**